pkg/lib/feature/accountdeletion/runnable.go

```diff
@@ -38,7 +38,8 @@ func (r *Runnable) Run(ctx context.Context) error {
 		return err
 	}
 	for _, appUser := range appUsers {
-		appContext, err := r.AppContextResolver.ResolveContext(ctx, appUser.AppID)
+		var appContext *config.AppContext
+		ctx, appContext, err = r.AppContextResolver.ResolveContext(ctx, appUser.AppID)
```

Comment:
We suffer the same project-specific-context-being-pass-to-next-iteration problem here.